07-routing-gossip.md

```diff
@@ -474,13 +479,13 @@ channel:
 C also uses a`min_final_cltv_expiry` of 9 (the default) when requesting
 payments.
 
-Also, each node has the same fee scheme which it uses for each of its
+Also, each node has a set fee scheme that it uses for each of its
 channels:
 
 1. A: 100 base + 1000 millionths
-1. B: 200 base + 2000 millionths
-1. C: 300 base + 3000 millionths
-1. D: 400 base + 4000 millionths
+2. B: 200 base + 2000 millionths
+3. C: 300 base + 3000 millionths
+4. D: 400 base + 4000 millionths
```

Comment:
This is fine, but FYI, markdown turns these into an enumerated list when rendering anyway :)